index.js

```diff
@@ -118,33 +118,28 @@ function blobLength (entry) {
   return entry.value.blob ? entry.value.blob.byteLength : 0
 }
 
-function pipeAll (rs, ws, transforms = []) {
+function pipeline (rs, ws) {
   return new Promise((resolve, reject) => {
-    try {
-      const onError = (err) => { if (err) reject(err) }
-
-      let p = rs
-      if (p && p.on) p.on('error', onError)
-
-      for (const t of transforms) {
-        if (t && t.on) t.on('error', onError)
-        p = p.pipe(t)
-      }
-
-      const dest = p.pipe(ws)
-      const finalStream = dest || ws
+    let done = false
+    const ondone = (err) => {
+      if (done) return
+      done = true
+      try { rs.unpipe(ws) } catch {}
```

Comment:
unpipe is not a thing outside node, if you want to destroy the stream, use .destroy()

Reply:
Barely hanging on here but I think we can just do without?